client/src/js/samples/components/Create/ReadSelector.js

```diff
@@ -8,7 +8,7 @@
  *
  * @exports ReadSelector
  */
-import { filter, includes, indexOf, intersection, map, sortBy, toLower, without } from "lodash-es";
+import { filter, includes, indexOf, isEqual, intersection, map, some, sortBy, toLower, without } from "lodash-es";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['some' is defined but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4748572)